### ISSUE TITLE
Allow field lineout in addition to slicing?

### ISSUE THREAD

**Issue by MaxThevenet** [openPMD/openPMD-viewer#193]
At the moment, users can get a slice of a field with `get_field(slicing=0., slicing_dir='y')`. Is there a way to get field lineouts? Do you think it could also be an option in OpenPMD? 

I was considering something like `get_field(lineout={'x':0., 'y':0.}, lineout_dir='z')`, or `get_field(lineout={'x':0., 'y':0.})` to avoid redundancy.
I see two ways to do it:
- change functions `read_field_3d`, `read_field_2d` and `read_field_circ` so that they take a new argument
- change `get_field` only so that it takes a slice and then extracts a lineout.

Do you have an opinion?

**Comment by RemiLehe** [comment]
I agree that it would be great to have the possibility to easily to lineouts.

If possible it would be great if the lineout API is compatible with the `slicing` API. For instance, what is your opinion about doing using:
```
ts.get_field( slicing=[0., 0.], slicing_dir=['x', 'y'] )
```
in order to get a lineout from 3D data?

**Comment by MaxThevenet** [comment]
Sure, that's great.
